### PR TITLE
Add book login 

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the new login feature. Its a new view that redirects to the index when you press the login button.

**New features**

- Set the form for the login
- Set the buttons with flex.
- Set the inputs.
- Set the Wolox logo over the form. 
- The login button now redirect you to the index, where the book list is. 

## Screenshots 
Screenshot from 2020-04-06 12-55-10

## Cards

https://trello.com/c/XXDIrY65/32-login

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="login-container">
+<div class="login-inputs">
```
Comment (eledwinn) [inline login.html]: Please indent all file :D

File: login.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                <input class="form-input" type="text" id="password" name="password"><br><br>
+                <button class="form-button-login" type="submit">Login</button>
+                <button class="form-button-singup" type="submit">Sing Up</button>
+              </form>
+    </div>
+</div>
+  </div>
+</body>
+
+</html>
```
Comment (eledwinn) [inline login.html]: Remember empty line at EOF

File: login.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+</head>
+
+<body>
+  <div class="login-container">
+<div class="login-inputs"> 
+    <img class="logo-wolox-login" src="images/LogoWolox-Original.png" alt= "Wolox Logotype">
+    <div class="form-container"> 
+            <form class="form-login" action="/index.html" id="form-login">
+                <label class="form-label" for="email">Email</label>
+                <input class="form-input" type="text" id="email" name="email"><br><br>
```
Comment (eledwinn) [inline login.html]: Avoid to use <br> for split controls... Use margin
For this, try insert each label and input into a div

File: login.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+                <label class="form-label" for="password">Password</label>
+                <input class="form-input" type="text" id="password" name="password"><br><br>
+                <button class="form-button-login" type="submit">Login</button>
+                <button class="form-button-singup" type="submit">Sing Up</button>
```
Comment (eledwinn) [inline login.html]: Only one button should be type="submit".

File: scss/detail.scss [hunk 5]
```diff
@@ -172,3 +172,72 @@
...
+  width: 252px;
+
+  &::before {
+    background: $silver-chalice;
+    content: "";
+    display: flex;
+    height: 2px;
+    position: relative;
+    top: 40px;
+    width: 252px;
```
Comment (eledwinn) [inline scss/detail.scss]: Try don't repeat `width` property, this should use same that button.

File: scss/detail.scss [hunk 6]
```diff
@@ -172,3 +172,72 @@
...
+
+.form-button-singup {
+  background-color: $wild-sand;
+  border: 2px solid $earls-green;
+  border-radius: 5px;
+  box-sizing: border-box;
+  color: $earls-green;
+  height: 32px;
+  width: 252px;
+}
```
Comment (eledwinn) [inline scss/detail.scss]: Try don't repeat same properties for both buttons. Props such as height, width, border-radius can join.

File: scss/detail.scss [hunk 7]
```diff
@@ -172,3 +172,72 @@
...
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
+}
+
+.logo-wolox-login {
+  height: 70px;
+  margin-bottom: 15px;
+  width: 215px;
+}
```
Comment (eledwinn) [inline scss/detail.scss]: Insert empty line here

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: login.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="login-container">
+<div class="login-inputs"> 
+    <img class="logo-wolox-login" src="images/LogoWolox-Original.png" alt= "Wolox Logotype">
```
Comment (CrisVega08) [inline login.html]:
```suggestion
    <img class="logo-wolox-login" src="images/LogoWolox-Original.png" alt= "Wolox Logotype" />
```

File: login.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+    <div class="form-container"> 
+            <form class="form-login" action="/index.html" id="form-login">
```
Comment (CrisVega08) [inline login.html]: check indentation

File: login.html (hunk 3 above)
Comment (CrisVega08) [inline login.html]:
```suggestion
                <input class="form-input" type="text" id="email" name="email">
```

File: scss/detail.scss [hunk 10]
```diff
@@ -172,3 +172,72 @@
...
+  border-radius: 5px;
+  color: $white;
+  height: 32px;
+  margin-bottom: 35px;
+  width: 252px;
+
+  &::before {
+    background: $silver-chalice;
+    content: "";
+    display: flex;
```
Comment (CrisVega08) [inline scss/detail.scss]: Why display flex ?

File: scss/detail.scss [hunk 11]
```diff
@@ -172,3 +172,72 @@
...
+    height: 2px;
+    position: relative;
```
Comment (CrisVega08) [inline scss/detail.scss:210]: Use position relative to .form-button-login no viewport

File: scss/detail.scss [hunk 12]
```diff
@@ -172,3 +172,72 @@
...
+.form-login {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-input {
+  background-color: $white;
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
```
Comment (CrisVega08) [inline scss/detail.scss]: you use a lot this value use on principal container and for other use with relative to this.

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 13]
```diff
@@ -172,3 +178,69 @@
...
+.login-container {
+  display: flex;
+  justify-content: center;
+}
+
+.form-button-login {
+  background-color: $earls-green;
+  color: $white;
+  justify-content: center;
+  margin: 25px 0 25px 0;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  margin: 25px 0;
```

File: scss/detail.scss [hunk 14]
```diff
@@ -172,3 +178,69 @@
...
+  margin-top: 10px;
+}
+
+.form-container {
+  display: flex;
+  justify-content: center;
+}
+
+.form-label {
+  margin: 20px 0 10px 0;
```
Comment (CrisVega08) [inline scss/detail.scss]:
```suggestion
  margin: 20px 0 10px;
```

File: login.html [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="login-container">
+    <div class="login-inputs">
+      <img class="logo-wolox-login" src="images/LogoWolox-Original.png" alt="Wolox Logotype" />
+      <div class="form-container">
+        <form class="form-login" action="/index.html" id="form-login">
```
Comment (CrisVega08) [inline login.html:18]: I remember a edwin's comment  about use a div something like:

```
<div class="form-group">
  <label class="form-label" for="email">Email</label>
  <input class="default-rectangle form-input" type="text" id="email" name="email">
</div>

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: login.html [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+          <label class="form-label" for="email">Email</label>
+          <input class="default-rectangle form-input" type="text" id="email" name="email">
```
Comment (eledwinn) [inline login.html]: Remember close inline tags with `/>`

File: login.html [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+          <label class="form-label" for="password">Password</label>
+          <input class="default-rectangle form-input" type="text" id="password" name="password">
+          <button class="default-rectangle form-button-login" type="submit">Login</button>
+          <button class="default-rectangle form-button-singup">Sing Up</button>
```
Comment (eledwinn) [inline login.html]: Buttons without `type` attribute have `submit` by default. Please always add `type="button"` or `type="submit"`

File: scss/detail.scss [hunk 18]
```diff
@@ -133,6 +133,12 @@
   justify-content: center;
 }
 
+.default-rectangle {
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
```
Comment (eledwinn) [inline scss/detail.scss]: I think the inputs should not be fixed width. Always try to use relative measures for the contents.
For example, if you change the width of the login box, will you also have to change the width of the inputs.

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: login.html [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+          </div>
+          <button class="default-rectangle form-button-login" type="submit">Login</button>
+          <button class="default-rectangle form-button-singup" type="button">Sing Up</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</body>
+
+</html>
```
Comment (CrisVega08) [inline login.html]: Add empty line

File: login.html [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+            <input class="default-rectangle form-input" type="text" id="password" name="password" />
+          </div>
+          <button class="default-rectangle form-button-login" type="submit">Login</button>
+          <button class="default-rectangle form-button-singup" type="button">Sing Up</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</body>
+
```
Comment (CrisVega08) [inline login.html]: delete it

File: scss/detail.scss [hunk 21]
```diff
@@ -172,3 +177,78 @@
...
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  flex: 1;
```
Comment (CrisVega08) [inline scss/detail.scss:233]:
```
flex: 1 equals = 
    flex-grow: 1;
    flex-shrink: 1;
    flex-basis: 0%;
```
So delete unnecessary property.

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**